meeting_minutes/220825_SRAIF_TC_65.md

```diff
@@ -0,0 +1,158 @@
+# 1. Opening Activities
```

Comment:
PR title should have 22-08-25, same as the file name fix

Reply:
... same as the event date (I hope we do not relate PR titles to files 😸 )